docs/book.toml

```diff
@@ -9,7 +9,7 @@ create-missing = false
 
 [output.html]
 additional-css = ["glean.css", "mermaid.css"]
-additional-js = ["tabs.js", "mermaid.min.js", "mermaid-init.js"]
+additional-js = ["tabs.js", "mermaid.min.js", "mermaid-init.js", "chart.min.js", "chart-histogram.js"]
```

Comment:
Is there a way to add these JS files only to the histogram simulator? I know I can just add the script tags directly, but is there a proper way?

Reply:
Nope, mdbook doesn't have per-chapter configuration